Birthday-2024-Project/Scripts/UI/StaticControlNode.gd

```diff
@@ -1,21 +1,53 @@
 extends Control
 class_name StaticControlNode
 
+@export var updateDelay : float = 0.5
+
 var _myParent : Control
-var _startingPosition : Vector2
+var _timer : float
+var _mousedOver : bool
+var _lastPosition : Vector2
+var _lastRotation : float
+
+func _onMouseEnter():
+	_mousedOver = true
+
+func _onMouseExit():
+	_mousedOver = false
 
 func _initialize():
```

Comment:
It probably sits somewhere here. Reset button arrow is visibly out of button for a split second every time level is opened
![image](https://github.com/Saplings-Projects/Birthday-2024/assets/150629422/3383b3fd-0bea-450a-b92a-fd1b8786d7fd)

Reply:
That looks and sounds like Godot animation messing with it. That screen is notoriously bad, so if you have it open in a tab, it has a chance to mess itself up

At least, the browser version is fine and it's fine in my project